utils/hare-start

```diff
@@ -40,4 +70,9 @@ if  hctl status > /dev/null 2>&1; then
     die 'Cluster is up and running.'
 fi
 
-hctl bootstrap -c $conf_dir
+if [[ $node ]]; then
+    hare-start-node
+else
+    hctl bootstrap -c $conf_dir
```

Comment:
So it means that if the user runs `hctl start` and forgets to put `--node`, the command will behave as `hctl bootstrap`. Why do we need this variance in `hctl start`? If the user wants to bootstrap the cluster, (s)he has a separate command.

Reply:
According to scripts, the purpose of `hctl start` (default) is to start all the nodes without regenerating the config files. 

`hctl bootstrap -c $conf_dir ` This part was already present in the script. Here, hctl bootstrap is called with `-c` arg, which **skips only the cfgen phase**. Rest of the steps present in bootstrap are followed as it is - like mk-consul-env, update-consul-conf (which I'm not sure whether its redundant or is there any specific use-case for doing so). 

I guess, we are just making the interface simpler for the user by adding this variant.

Reply:
@knekrasov , ideally node start, join and stop must be subcommands for `hctl node` command but because `hctl node` is still bound to `pcswrap`. We need to get rid of this last part remaining in hare with respect to HA. I will create a new ticket to sort this once and for all. Let's transfer the last remaining `pcswrap` piece as well to HA and update CSM to switch to HA interface instead of Hare. We can then free up `hctl node` for Hare specific commands.

Reply:
https://jts.seagate.com/browse/EOS-18322